File: src/Wowthing.Lib/Enums/ZoneMaps.cs
namespace Wowthing.Lib.Enums
{
    public enum FarmType
    {
        Kill = 1,
        KillBig,
        Puzzle,
        Treasure,
        Event,
        EventBig,
        Cloth,
        Leather,
        Mail,
        Plate,
        Weapon,
        Vendor,
        Group,
        Quest,
    }

    public enum FarmIdType
    {
        Npc = 1,
        Object,
        Quest,
    }
    
    public enum FarmResetType
    {
        None,
        Daily,
        BiWeekly,
        Weekly,
        Never,
    }

    public enum FarmDropType
    {
        Pet = 1,
        Mount,
        Quest,
        Toy,
        Cosmetic,
        Armor,
        Weapon,
        Achievement,
        Transmog = 100,
    }
}

File: apps/frontend/icons/zone-maps.ts
import { FarmType } from '@/enums'


export const farmTypeIcons: Record<number, string> = {
    [FarmType.Achievement]: 'gameAchievement',
    // [FarmType.Dungeon]: faDungeon,
    // [FarmType.Event]: mdiTimerSand,
    // [FarmType.EventBig]: mdiTimerSand,
    // [FarmType.Kill]: mdiSkull,
    [FarmType.KillBig]: 'gameCrownedSkull',
    // [FarmType.Profession]: mdiHammerWrench,
    // [FarmType.Puzzle]: mdiPuzzle,
    // [FarmType.Quest]: hisExclamationCircle,
    // [FarmType.Raid]: faDungeon,
    [FarmType.Treasure]: 'gameLockedChest',
    [FarmType.Vendor]: 'mdiCart',

    // [FarmType.Cloth]: mdiLetterC,
    // [FarmType.Leather]: mdiLetterL,
    // [FarmType.Mail]: mdiLetterM,
    // [FarmType.Plate]: mdiLetterP,
    // [FarmType.Weapon]: mdiAxeBattle,
}
